File: cpp/Assignment 1/Set A/a1.cpp
#include<iostream>
using namespace std;
int main(){
    int r,i,j,n=1;
    cout<<"Enter Number of Rows:";
    cin>>r;

    for(i=1;i<=r;i++){
        for(j=1;j<=i;j++){
            cout<<n<<" ";
            n++;
        }
        cout<<endl;
    }
    return 0;
}

File: php/Assignment 2/SET A/a1.php
<html?>
<body>
    <style>
              body {
                  background-color:skyblue;
              }
              h1 {text-align: center;}
        </style>
    <form  method="GET">
        <label for="num">Enter number</label>
        <input type="text" name="num" id="num"> 
        <input type="submit" name="submit" id="">
    </form>
    <?php 
        $num = $_GET['num'];
        $factorial = 1;
        for ($x=$num; $x>=1; $x--)
        {
        $factorial = $factorial * $x;
        }
        echo "Factorial of $num is $factorial";
        ?>
</body>
</html>